server/models/draftReferral.ts

```diff
@@ -23,8 +23,9 @@ export interface ReferralFields {
   whenUnavailable: string | null
   serviceUser: ServiceUser
   additionalRiskInformation: string
-  usingRarDays: boolean
-  maximumRarDays: number | null
+  usingRarDays: boolean // deprecated
+  maximumRarDays: number | null // deprecated
+  maximumEnforceableDays: number | null
```

Comment:
I reckon this will no longer be nullable, now that the user has to enter a number of days?

Reply:
Doh, I was getting confused with where it starts out as `null` but that's where we extend it with nullable fields. Will fix this, thanks.